firestore/FirestoreSwiftUIExample/Views/ContentView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  ContentView.swift
+//  FirestoreSwiftUIExample
+//
+
+import SwiftUI
+import Firebase
+
+struct ContentView: View {
+    let db = Firestore.firestore()
```

Comment:
I'd generally refrain from directly accessing the backend from a view. Instead, let's try to educate our users to always use view models.